5-infrastructure/modules/base-env/boa-sql.tf

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+locals {
+  sql_settings = {
+    sql_1 = {
+      database_zone        = "${var.location_primary}-a",
+      database_name        = "ledger-db",
+      database_users       = [],
+      additional_databases = [],
+      replica_zones = {
+        zone1 = "${var.sql_database_replication_region}-a",
+        zone2 = "${var.sql_database_replication_region}-c",
+        zone3 = "${var.sql_database_replication_region}-f"
+      }
+      encrypt_keyring_name = module.kms_keyrings_keys["kms_sql_1"].keyring_name,
+      sql_instance_prefix  = "sql-boa-${var.location_primary}-01",
+      database_region      = var.location_primary
+    },
+    sql_2 = {
+      database_zone        = "${var.location_secondary}-a",
+      database_name        = "accounts-db",
+      database_users       = [],
```

Comment:
If these are `[]` for both, we can just set it in the module?

Reply:
allowing the user to set them at the base_env level for convinience